packages/design-system/src/components/InlineEditing/InlineEditing.tsx

```diff
@@ -133,19 +136,22 @@ const InlineEditing = React.forwardRef(
 						onDoubleClick={loading ? undefined : () => setEditMode(true)}
 					>
 						<S.InlineEditingValue
-							className="c-inline-editing__value"
+							className={classNames('c-inline-editing__value', { placeholder: !value })}
+							data-text={placeholder}
 							as={renderValueAs || renderAs}
+							aria-hidden
```

Comment:
Should be aria-hidden *only* if there is no value
```suggestion
							aria-hidden={value !== undefined}
```